iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -23,6 +23,12 @@
  */
 public final class ValidationUtil {
 
+    public enum LogLevel {
```

Comment:
Instead of `LogLevel`(that could be confused with the logger behavior), I prefer `ActionOnError` for this enum name.

Reply:
Done